oraichain/assetlist.json

```diff
@@ -504,6 +528,10 @@
       "symbol": "wETH",
       "images": [
         {
+          "image_sync": {
+            "chain_name": "ethereum",
+            "base_denom": "weth"
```

Comment:
instead of weth image (that's not the correct base for WETH, anyway), I recommend just using the eth logo image (base: "wei")